### ISSUE TITLE
Add schema identificator to failed validation message

### ISSUE THREAD

**Issue by chuwy** [snowplow/iglu-scala-client#73]
Right now it just gives information on why instance is invalid: 

```
error: object has too few properties (found 0 but schema requires at least 1)
    level: ""error""
    schema: {""loadingURI"":""#"",""pointer"":""""}
    instance: {""pointer"":""""}
    domain: ""validation""
    keyword: ""minProperties""
    found: 0
    required: 1
```

But doesn't say *which* instance is invalid and Support needs to make guesses where is the error exactly. This should be:

```
error: instance of "iglu:com.acme/event/jsonschema/1-0-2": object has too few properties (found 0 but schema requires at least 1)
    level: ""error""
    schema: {""loadingURI"":""#"",""pointer"":""""}
    instance: {""pointer"":""""}
    domain: ""validation""
    keyword: ""minProperties""
    found: 0
    required: 1
```

**Comment by alexanderdean** [comment]
This would be a huge win. It would be even better if the instance was a first-class field, not embedded in a string, to enable easy analytics. /cc @mikenem

**Comment by chuwy** [comment]
It can be problematic with our current approach, using fge's `ProcessingMessage` type, but worth to try, agree.

**Comment by chuwy** [comment]
This was implemented as part of #113 and #78
